src/index.js

```diff
@@ -1,18 +1,37 @@
-import _ from 'lodash';
 import './style.css';
+import Icon1 from './kebab.png';
+import Icon2 from './refresh.svg';
 
+const tasks = [
+  { description: 'eat', completed: true, index: 0 },
+  { description: 'code', completed: true, index: 1 },
+  { description: 'sleep', completed: false, index: 2 },
+];
 
-
-function component() {
+const headline = document.querySelector('.headline');
+const refreshIcon = new Image();
+refreshIcon.src = Icon2;
+refreshIcon.className = 'refresh-image';
+headline.appendChild(refreshIcon);
+function CreateList(tasks) {
+  for (let i = 0; i < tasks.length; i += 1) {
+    const list = document.querySelector('.dynamic-list');
     const element = document.createElement('div');
-  
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+    element.className = 'bg task';
+    /* eslint-disable */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.